associated-token-account/program-test/Cargo.toml

```diff
@@ -16,4 +16,4 @@ solana-program-test = "1.16.3"
 solana-sdk = "1.16.3"
 spl-associated-token-account = { version = "2.0", path = "../program", features = ["no-entrypoint"] }
 spl-token = { version = "4.0", path = "../../token/program", features = ["no-entrypoint"] }
-spl-token-2022 = { version = "0.7", path = "../../token/program-2022", features = ["no-entrypoint"] }
+spl-token-2022 = { version = "0.8", path = "../../token/program-2022", features = ["no-entrypoint"] }
```

Comment:
I don't think we have a reason to release it, do we?

Reply:
Maybe not. Bumping one of it's dependencies isn't grounds to release it? Because no code has changed within it right? As long as we're not re-exporting token 2022 then it's cool

Reply:
oh wait never mind, you're absolutely right. token-client and token-cli use associated-token-account

Reply:
side note: this is why just having everything on the same version and releasing them as one simplifies a lot